volttrontesting/platform/dbutils/test_postgresqlfuncts.py

```diff
@@ -0,0 +1,880 @@
+import contextlib
+import datetime
+
+from time import time
+
+from psycopg2.sql import SQL, Identifier
+import pytest
+import psycopg2
+
+from volttron.platform.dbutils.postgresqlfuncts import PostgreSqlFuncts
+from volttrontesting.fixtures.docker_wrapper import create_container
+from volttrontesting.utils.utils import get_rand_port
+
+# Current documentation claims that we have tested Historian on Postgres 10
+# See https://volttron.readthedocs.io/en/develop/core_services/historians/SQL-Historian.html#postgresql-and-redshift
+# Since we are using TravisCI, running these tests on more than three images will stop the build because it takes more than 10 minutes to produce any log output.
+# See https://docs.travis-ci.com/user/customizing-the-build/#build-timeouts
+# Therefore, a maximum of two images are used for testing on TravisCI.
+# However, there are no limits to run this test locally with all images.
+# To run tests on all images, uncomment the commented images and run `./ci-integration/virtualization/core/entrypoint.sh'
+IMAGES = [
+    "postgres:9.6.18",
+    "postgres:10.13",
+    # "postgres:11.8",
```

Comment:
Rather than doing this, can you read the environment variable TRAVIS_CI to determine if it is running on travis and then only run the first two in that case or better yet, extend the IMAGES with the others when not travis_ci.  See https://docs.travis-ci.com/user/environment-variables#default-environment-variables for default env variables.

Reply:
Using the env vars for test config makes total sense. I have made the changes and tested them on my throwaway test branch on Travis: see https://travis-ci.org/github/bonicim/volttron/builds/714970732